lib/system.nim

```diff
@@ -46,7 +46,7 @@ type
 
   void* {.magic: "VoidType".}    ## Meta type to denote the absence of any type.
   auto* {.magic: Expr.}          ## Meta type for automatic type determination.
-  any* = distinct auto           ## Meta type for any supported type.
+  any* {.deprecated: "Deprecated since v1.5; Use auto instead.".} = distinct auto  ## Deprecated; Use `auto` instead. See https://github.com/nim-lang/RFCs/issues/281
```

Comment:
can you try to outright remove `any`, see what breaks (both in nim repo and in important_packages), fix the resulting `any` to `auto` so that stdlib is purged from it, and then re-enable the (now deprecated) `any` ? that's a bit of work but also needed to assess damage and avoid generating new warnings (and also what I try to do in my other PR's that deprecated X)

Reply:
stdlib and compiler works Ok.
Some package uses `any`.
But this meant to Deprecate, not remove.

Reply:
> Some package uses any.

did you note which ones?

Reply:
https://github.com/nim-lang/Nim/runs/1823138277
chronicles, telebot, etc